charger/heidelberg-ec.go

```diff
@@ -108,15 +119,24 @@ func (wb *HeidelbergEC) Status() (api.ChargeStatus, error) {
 		return api.StatusNone, err
 	}
 
-	switch sb := b[1]; sb {
+	switch wb.laststatus = binary.BigEndian.Uint16(b); wb.laststatus {
 	case 2, 3:
 		return api.StatusA, nil
 	case 4, 5:
 		return api.StatusB, nil
 	case 6, 7:
 		return api.StatusC, nil
+	case 8:
+		return api.StatusD, nil
+	case 9:
+		return api.StatusE, nil
+	case 10:
+		// Exit the fail-safe mode and return to normal operation after wakeup
```

Comment:
Das gehört nixht in die Statusanfrage

Reply:
Am liebsten würde ich in dem Fall gar keinen (neuen) Status zurückgeben. Geht das?

Reply:
ich würde den Wakeup laufen lassen bis es einen neuen Status gibt oder einfach Delay. Die Statii hier brauchts nicht da evcc damit eh nix macht. Alles ausser A-C ist ein Fehler für uns.

Reply:
Status F wird ja hier momentan als asynchroner Indikator genutzt dass der Timeoutmodus aktiv ist und wieder beendet werden muss.
Meinst du man sollte das besser in der Wakeup-Funktion mittels aktivem Delay lösen?
Das würde im schlimmsten Fall über 10s Pause und Blockade bedeuten. Da hab ich erstmal ein Angst vor. Kannst du mir die nehmen?

Reply:
es gibt nichts das an dieser Stelle auf Status F reagieren würde

Reply:
`Case 10` ist Status F

Reply:
Die Logik einen Status zu verlassen gehört hier einfach nicht hin!

Reply:
Wohin in dem Fall denn? Vorschlag?

Reply:
Da wo der Ärger verbrochen wird. Also nach https://github.com/evcc-io/evcc/pull/2287/files#diff-e995718201ef1d5e24ad9973bca73288ed812aca6bae2d6b97fbcc3acee99e04R269